src/Framework/Conventions/KinePhaseSpace.h

```diff
@@ -64,6 +64,7 @@ typedef enum EKinePhaseSpace {
   kPSTkTlctl,
   kPSQ2vfE,
   kPSQELEvGen,// Phase space used by genie::QELEventGenerator for sampling kinematic variables
+  kPSDMELEvGen,// Equivalent for Dark Matter scattering
               // TODO: rename this value when the correct variables are identified
   kPSTAfE
 } KinePhaseSpace_t;
```

Comment:
Does that enumeration need to go at the bottom for backward compatibility? (at least, till we break connection with GENIE3 for good in GENIE4?) Need comment from @nusense

Reply:
Yes, all additions to enums should go at the end of the list lest one introduce a skew between files written with an earlier version (output is stored as a _value_ not an _enum tag_) but are subsequently read in by a later version.   Enums are great for _code_ but lacking in how the map to values in i/o.